### ISSUE TITLE
Property 'find' does not exist on type 'number error on nmp start.

### ISSUE THREAD

**Issue by msooraj** [coursefiles/angular2-essential-training#8]
Getting "Property 'find' does not exist on type 'number" error on npm start.

Please set the compiler target to es6 in your tsconfig.

**Comment by jschwarty** [comment]
Looks like it's an issue with a newer version of `@types/core-js`. The `package.json` file is currently set up to use the `^` on that package and the `@types/node` package which results in you getting the latest version of those. I am going to update the code in the repo to fix that, but for the meantime you can update your local package file like so (removing the ^ from in front of the version number for these two):

```json
"devDependencies": {
    "@types/core-js": "0.9.34",
    "@types/node": "6.0.41",
    ...
  }
```
Then you need to delete your `node_modules` directory and run `npm install` again.

**Comment by jschwarty** [comment]
Okay, all the branches have been updated to remove the caret on the `@types` packages and fix this issue. Thanks @msooraj for letting me know about this issue! I appreciate you reporting it.

**Comment by msooraj** [comment]
Thanks for fixing. Just started the course in Linda. It is very helpful.

**Comment by WARPed1701D** [comment]
Hi.  I just cloned the course files and while in the MASTER branch I ran npm install and npm start. I received the same error on running npm start as originally reported. Removing the ^ from packages.json as described above resolved the issue. Looking at the packages file in the master branch on GitHub shows no ^. Yet I cloned the repo again and sure enough in the packages.json file the ^ was there for both core-js and node.

![caret](https://cloud.githubusercontent.com/assets/4031283/25101895/0ef0a476-2384-11e7-91e9-8629d40a886d.png)
 
![caret](https://cloud.githubusercontent.com/assets/4031283/25101981/65663046-2384-11e7-98b4-56b07646c2b7.png)
